.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,72 @@
+name: Bug report
+description: Create a report to help us improve the project
+title: ''
+labels: #bug
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Steps to reproduce the behavior
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+        4. See error
+    validations:
+      required: true
+  - type: dropdown
+    id: device-type
+    attributes:
+      label: Device type
+      description: What type of device are you using?
+      multiple: false
+      options:
+        - label: Desktop
+        - label: Smartphone
```

Comment:
Is it even useful here? It's now like Hexa is a mobile app... and "Platform" has same info

Reply:
@PeyTy okay, fixed already